pkg/project/client.go

```diff
@@ -211,9 +212,20 @@ func getClient(ctx context.Context, cfg *config.CLIConfig, opts Options, project
 		New: func() (client.Client, error) {
 			url := cfg.TestProjectURLs[serverOrKubeconfig+"/"+account]
 			if url == "" {
-				url, err = manager.ProjectURL(ctx, serverOrKubeconfig, account, cred.Password)
-				if err != nil {
-					return nil, err
+				loginRetry := true
+				for {
+					url, err = manager.ProjectURL(ctx, serverOrKubeconfig, account, cred.Password)
+					if loginRetry && errors.Is(err, manager.ErrForbidden) {
+						cred.Username, cred.Password, err = manager.Login(ctx, cfg, "", serverOrKubeconfig)
+						if err != nil {
+							return nil, err
+						}
+						loginRetry = false
+						continue
+					} else if err != nil {
+						return nil, err
+					}
+					break
 				}
```

Comment:
@thedadams could you explain the intended behavior here? It looks like the `for` loop is never executed more than once.

Reply:
The for loop is executed at most twice.

If the ProjectURL call fails with a "forbidden" error on the first try, then we login, set `loginRetry` to false and `continue`. If the ProjectURL call fails for any reason in the second try, then we return the error.

If all errors are nil, then we get to the break.

Reply:
Ah, I see it now. Thanks!